dna/zomes/courses/code/src/course.rs

```diff
@@ -21,119 +21,223 @@ use std::convert::TryFrom;
 /******************************************* */
 
 #[derive(Serialize, Deserialize, Debug, DefaultJson, Clone)]
-pub struct Course {
+pub struct CourseData {
+    // This struct's instances would be updated if we need to change anything about the course
     pub title: String,
     pub modules: Vec<Address>,
     pub timestamp: u64,
     pub teacher_address: Address,
```

Comment:
Is the teacher address correct to store with the CourseData?

Reply:
There are 2 reasons for that:
1) teacher address improves chances of every particular CourseData entry being unique
2) since CourseData contains everything describing the course from the user perspective, we need to include teacher address as well